src/components/Conditional.js

```diff
@@ -21,6 +21,8 @@ const Conditional = (props) => {
     isFalse: node.text === node.data.get('whenFalse'),
   };
 
+  const emptyConditionalFlag = node.text === conditional.whenTrue;
```

Comment:
Still curious about this. There are two types of conditionals: 
1. `whenTrue = 'something'` and `whenFalse = ''`
2. `whenTrue = 'something'` and `whenFalse = 'something else'`

So the code right now will not have `className='conditional'` when 2 is in false state. 

If `conditional.value` is undefined in this scope and `node.text` is the value we're looking for...  We're looking to see if `node.text === conditional.whenFalse` if `conditional.whenFalse === ''`, correct? Then in the `conditionalProps` you could just replace `emptyConditionalFlag` with  `node.text === ''` right?

Reply:
Yes @irmerk We can do this ! I just checked it works fine.
I guess ```whenTrue``` is the text to display when the conditional is not hidden and ```whenFalse``` is the one to show when it is. Am I right?
So I was using ```whenFalse``` and ```whenTrue``` for that.
Should I make this change and push?

Reply:
This is how it is looking now 
```js
const conditionalProps = {
    id: conditional.id,
    className: node.text === '' ? '' : 'conditional',
    onMouseEnter: () => setHovering(true),
    onMouseLeave: () => setHovering(false),
    onClick: () => { toggleConditional(node.key); },
    ...attributes,
  };
```

Reply:
> I guess whenTrue is the text to display when the conditional is not hidden and whenFalse is the one to show when it is. Am I right?

Mostly. That's my number 1 scenario. There is number 2 though, where `whenFalse` doesn't need to be an empty string.

Reply:
Yes I think that looks good to me.

Reply:
Agreed. `className: node.text === '' ? '' : 'conditional'` seems like the right approach to me!